packages/datadog-plugin-aws-sdk/src/services/bedrockruntime.js

```diff
@@ -0,0 +1,254 @@
+'use strict'
+
+const BaseAwsSdkPlugin = require('../base')
+const log = require('../../../dd-trace/src/log')
+
+const AI21 = 'AI21'
+const AMAZON = 'AMAZON'
+const ANTHROPIC = 'ANTHROPIC'
+const COHERE = 'COHERE'
+const META = 'META'
+const STABILITY = 'STABILITY'
+const MISTRAL = 'MISTRAL'
+
+const enabledOperations = ['invokeModel'] // TODO check if this is the correct operation
```

Comment:
```suggestion
const enabledOperations = ['invokeModel']
```
we should be good here now!